index.bs

```diff
@@ -1344,11 +1354,13 @@ partial interface HTMLIFrameElement {
     And in the same section, in step 10, set the new {{Document}}'s
     [=Document/report-only permissions policy=] to |reportOnlyPermissionsPolicy|.
 
-    And in the same section, in step 19 before the return, insert the following step:
+    In <a
```

Comment:
This implies that this check only applies to iframe elements, and not to any other kind of embed. (<fencedframe>, <frame>, <object>, <embed>, etc.) Is that correct?

Reply:
Right, because `allow` attribute is only present in iframe elements.

Reply:
Thanks for confirming!